portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -0,0 +1,54 @@
+ 
+ 
+package com.google.sps.servlets;
+
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+
+@WebServlet("/list-post")
+public class ListCommentServlet extends HttpServlet {
+ 
+ @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.setContentType("application/json;");
+    Query query = new Query("Messages").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
```

Comment:
I've noticed you are not using gson to convert these to entity properties to json. That should happen before you do response.getWriter().println(). Not sure if that is the issue you are experiencing though.

Reply:
Sorry for the inconvenience on my previous commit. If I need to use gson, I am now trying to use ArrayList to temporarily store data for displaying. I will get back to you soon.